bambi/tests/test_built_models.py

```diff
@@ -338,37 +350,23 @@ def test_laplace_regression():
     bmb_model.fit()
 
 
-def test_logistic_regression_numpyro():
-    y = pd.Series(np.random.choice(["a", "b"], 50), dtype="category")
-    data = pd.DataFrame({"y": y, "x": np.random.normal(size=50)})
-    model = Model("y ~ x", data, family="bernoulli")
+def test_logistic_regression_numpyro(logistic_regression_data):
```

Comment:
These next couple could be parametrized so now theres only one test. We're really squeezing code lines out now but I hope its a good learning experience!